_rules/video-only-description-track-ac7dc6.md

```diff
@@ -87,8 +87,8 @@ A video only element that is not visible on the page.
 
 ```html
 <video controls style="display: none;">
-	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4" />
-	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm" />
+	<source src="../test-assets/rabbit-video/silent.mp4" type="video/mp4">
+	<source src="../test-assets/rabbit-video/silent.webm" type="video/webm">
 	<track kind="descriptions" src="rabbit-video-descriptions.vtt" />
```

Comment:
`<track>` is also a void element, so it shouldn't have the end slash, similar to `<source>`